src/coinbase/types.ts

```diff
@@ -1362,3 +1362,20 @@ export interface SmartContractAPIClient {
     options?: RawAxiosRequestConfig,
   ): AxiosPromise<SmartContractModel>;
 }
+
+/**
+ * Options for pagination on list methods.
+ */
+export type PaginationOptions = {
+  limit?: number;
+  page?: string | undefined;
```

Comment:
remove `| undefined` as it's already undefined from the `?`

Reply:
done